packages/atst/src/commands/write.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { Address } from '@wagmi/core'
+import { Wallet } from 'ethers'
+import { describe, expect, it } from 'vitest'
+
+import { ATTESTATION_STATION_ADDRESS } from '../constants/attestationStationAddress'
+import { read } from './read'
+import { write } from './write'
+
+describe(`cli:${write.name}`, () => {
+  it('should write attestation', async () => {
+    // Anvil account[0]
```

Comment:
Leftover? Not sure I understand what does it mean :D

Reply:
It's the first anvil/hardhat test account
<img width="572" alt="image" src="https://user-images.githubusercontent.com/35039927/221722028-0902d77d-52dc-432b-974c-196a32084550.png">

Reply:
updated this to be more clear btw